integration/presto/src/test/scala/org/apache/carbondata/presto/integrationtest/PrestoTestUsingSparkStore.scala

```diff
@@ -20,14 +20,15 @@ package org.apache.carbondata.presto.integrationtest
 import java.io.{File}
 import java.util
 
+import io.prestosql.jdbc.PrestoArray
```

Comment:
I think presto db compile will fail after this. can you handle validation in the util method like `PrestoTestUtil#validateArrayOfPrimitiveTypeData`

Reply:
Also please verify presto db compile once to catch if any other things broke compilation.

Reply:
done